src/features/identity/identity-agent.ts

```diff
@@ -0,0 +1,29 @@
+import { IdentityAgent } from "@web5/identity-agent";
+import { AppDataVault, DwnManager } from "@web5/agent";
+import { ExpoLevelStore } from "./expo-level-store";
+import { Dwn } from "@tbd54566975/dwn-sdk-js";
+
+export async function bootstrapIdentityAgent(
+  passphrase: string,
+  name: string,
+  dwn: Dwn
+) {
+  const dwnManager = new DwnManager({ dwn });
+  const appData = new AppDataVault({
+    keyDerivationWorkFactor: 1,
+    store: new ExpoLevelStore("AppDataVault"),
```

Comment:
is this a stopgap until we create a securestore for this?

Reply:
@frankhinek is there any plans here? afaik, this is the solution that was come up with, no idea about secure stores yet.

Reply:
I guess my question is are we supposed to handle encryption using the device's native libraries (e.g. using keychain) or is web5 internally handling encryption for us?

Reply:
According to @frankhinek's document, that encryption is already done before being put into the store: https://hackmd.io/ShKU_Q9aQC6C96v7SaXl3A?view#First-Launch

@frankhinek: lmk if that's not correct, and we need to do some additional encryption on top!